### DISCUSSION TITLE
How to get client's time zone

### DISCUSSION THREAD

**Discussion by lutuantai95** [js-fns/date-fns#3356] [Q&A]
Hi guys, 
I had difficulties to get current time zone as below:

`let timeZone = new Date() -> 'Mon Feb 27 2023 17:52:48 GMT+0700 (Indochina Time)'`

Does anyone know how to get `GMT+0700`  or `UTC+07` from date-fns ?

Thanks in advanced!

**Comment by gabrielbergoc** [comment]
JS `Date` class has a method called `getTimezoneOffset` that returns the client's offset (in minutes). Then you can make your own function to convert the output to your desired format.
Docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset

**Reply by lutuantai95** [reply]
Thanks for your helps! 
Yes, we could use `getTimezoneOffset` but i'd like to get time zone from date-fns for clear code 😄
